### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by dskloetd**
Add something about snapshots to the README.md.

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
Thanks!

Just one typo

File: README.md [hunk 1]
```diff
@@ -1,10 +1,34 @@
-# Local NNS and SNS
+# SNS DEMO
```
[the remark this answers is not in the document]
Comment (dskloetd) [reply README.md:1]: Maybe. But names are hard to change. I'm afraid we're stuck with this :-).

File: README.md [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
 
-Welcome to the world of the Network Nervous System.  
+This repository has a collection of tools to make it easier to manage
+environments with SNS projects and other dependencies. If this sounds vague it's
+because it's a bit of a kitchen sync. Some of them are described below.
```
[the remark this answers is not in the document]
Comment (dskloetd) [reply README.md]: Haha, thanks. "Kitchen sync" is the name of one of the Google cafeteria.
Done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply README.md:1]: Yeah, I didn't mean in this PR...

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply README.md]: That's a good name for a cafeteria!

**Pull request merged**